app/models/card.rb

```diff
@@ -6,6 +6,20 @@ class Card < ApplicationRecord
 
   REVIEW_TIME_GAP = 3
 
+  def self.random
+    return nil if due_today.empty?
```

Comment:
@davetoxa it does in rails console but I get this strange bug where it will return Card.all (in rails, not in console) if I use `where("review_date < ?", DateTime.now).order("RANDOM()").first`